Commented fifth line about build

GuizmoPhil opened the pull request:
I was unable to have the container start with `docker-compose up -d` without commenting this fifth line.

cturra commented:
doh! sorry you're having a hard time. i just double checked on my own setup and do not seem to be running into the same issue as you. can you share a little more about your environment and the full commands you're running (and the output you see)? this might be able to point us in the right direction to explain why.

to share more about my setup, here is what is working for me:

```
$> docker-compose --version
docker-compose version 1.29.2, build 5becea4c

$> docker-compose up -d ntp
Creating ntp ... done

$> docker-compose ps ntp
Name           Command              State                      Ports
------------------------------------------------------------------------------------
ntp    /bin/sh /opt/startup.sh   Up (healthy)   0.0.0.0:123->123/udp,:::123->123/udp

$> docker-compose logs ntp
Attaching to ntp
ntp    | 2021-10-06T05:38:16Z chronyd version 4.1 starting (+CMDMON +NTP +REFCLOCK +RTC +PRIVDROP -SCFILTER +SIGND +ASYNCDNS +NTS +SECHASH +IPV6 -DEBUG)
ntp    | 2021-10-06T05:38:16Z Disabled control of system clock
ntp    | 2021-10-06T05:38:24Z Selected source 162.159.200.123 (time.cloudflare.com)

```

GuizmoPhil commented:
Well, now it is indeed working... 🦗 

I'm not an advanced user with docker, so I'm not familiar with the `build:` option. I'm assuming it is used to pull a specific version of the container. But a single dot means what exactly? I've seen **latest** being used in the past, but the dot is new to me.

Maybe it wasn't working from the get go prior to pulling the container upon first startup.

Or maybe it has something to do with the damn yml file format, spaces are the upmost important thing. Maybe I deleted a _tab space_ of something from the original copy-paste. The way I created the docker-compose.yml file : I copied the docker-compose.yml text from GitHub that was opened in my browser and then I pasted it in a Terminal running nano.

The command I was using that wasn't working, and is now working: `docker-compose up -d`

I have one docker-compose.yml file per container, this is why I do not specify the service name in the above command.

My docker-compose version: **1.25.0, build unknown** (it's probably the one from LinuxMint 20 repo (based un Ubuntu 20.04 focal). Since it is an older version than yours, it could also be a problem from this version.

I'm running docker as per official documentation, I've removed any OS-repo/distro and installed from Docker's own repo following theses instructions: https://docs.docker.com/engine/install/ubuntu/#install-using-the-repository
My docker version is: **20.10.9, build c2ea9bc**

In any case, it seems that I cannot reproduce the problem now, but I must have spent 30 minutes or so trying to make to work until I decided to add the comment at the start of the fifth line. Very strange behavior.
